x/perpetual/client/cli/query_mtp_test.go

```diff
@@ -64,6 +64,7 @@ func networkWithMTPObjects(t *testing.T, n int) (*network.Network, []*types.MtpA
 			},
 			TradingAssetPrice: sdk.ZeroDec(),
 			Pnl:               sdk.ZeroDec(),
+			UpdatedLeverage:   sdk.ZeroDec(),
```

Comment:
@cryptokage1996 can you share a test result where the new `updated leverage` field return a positive value for an existing position thanks